pkg/bff/docs/docs.go

```diff
@@ -1333,6 +1339,9 @@ const docTemplate = `{
                 "email": {
                     "type": "string"
                 },
+                "expires_at": {
+                    "type": "string"
+                },
```

Comment:
The files in `docs` are autogenerated so treat them as `.pb` files for the purposes of review.